PFS-IPMU-00001/startdown.rst

```diff
@@ -0,0 +1,51 @@
+Service cold start and full shutdown procedures
+------
+
+This memo shows ways to start all from cold and to shutdown all, with points 
+in consideration or taken cared of.
+
+Cold start
+======
+
+Start backends first, and end point to external. 
+
+* Power on storage server
+
+  * iSCSI server need to be started from JBOD, and controller follows
+  * Just power on SAS storage box
+
+* Power on VM host for NFS servers (dlb8-vm)
+* Start VM guests of NFS servers (pfsdisk, pfscalc, simdata; no order)
+* Start core management host (mgmt-dhcp, including DHCP, LDAP)
+* Start backend services
+
+  * DB
+  * rsyslog
+  * Elasticsearch (log target of rsyslog)
+
+* Start service VM guests
+
+  * jirapipe
+  * landfill
+
+* Start external servers (pfs, pfspipe, pfsarch)
+* Start monitoring services (prometheus, influxdb)
+* Start internal services
+
```

Comment:
After it, wee had to start jira service manually.
I think you will check it, but could you please add this step to procedure for now?

Reply:
init/systemd script by jira package does not support enabling automatically. it is better to be written, so added.

Reply:
Thanks.